device/components/RTA_OnDeviceComponent.brs

```diff
@@ -549,26 +549,222 @@ function processStoreNodeReferencesRequest(args as Object) as Object
 	return result
 end function
 
-sub buildTree(storedNodes as Object, flatTree as Object, node as Object, searchForArrayGrids as Boolean, arrayGridNodes = {} as Object, parentRef = -1 as Integer, position = -1 as Integer)
+' ArrayGrid children can't be built with a normal call to buildTree since you can only get the parent not the children
+' Often times nodes are in different spots as well and we want to standardize that as well to a consistent format
```

Comment:
```suggestion
' ArrayGrid children can't be built with a normal call to buildTree since you can only get the parent not the children.
' Often times nodes are in different spots, so this will also standardize them to a single consistent spot
```